src/Bicep.LangServer.IntegrationTests/CodeActionTests.cs

```diff
@@ -82,32 +82,6 @@ public async Task RequestingCodeActionWithFixableDiagnosticsShouldProduceQuickFi
             }
         }
 
-        [DataTestMethod]
-        [DynamicData(nameof(GetData), DynamicDataSourceType.Method, DynamicDataDisplayNameDeclaringType = typeof(DataSet), DynamicDataDisplayName = nameof(DataSet.GetDisplayName))]
-        public async Task RequestingCodeActionWithNonFixableDiagnosticsShouldProduceEmptyQuickFixes(DataSet dataSet)

```

Comment:
The test is removed because the assumption is wrong...apparently there can be one fixable diagnostic and one not fixable diagnostic overlapping with each other.